test/schemas.ts

```diff
@@ -3286,6 +3286,14 @@ export const v210Schema = {
   },
 }
 
+export const v211Schema = {
```

Comment:
@finnian0826 Just noticed this when attempting to add a new schema in a separate PR, but I think there's a missing schema here.

```
...v210Schema,
..._.omit(v210Schema, 'exchange'),
  _persist: {
    ...v210Schema._persist,
    version: 211,
  },
}